From e571bbfef0a988deeec97512441c9504e3042117 Mon Sep 17 00:00:00 2001
From: Pantelis Antoniou <panto@antoniou-consulting.com>
Date: Mon, 22 Apr 2013 20:38:11 +0300
Subject: [PATCH 06/11] bone-eMMC: Add rstctl rstctl DT bindings

Add the references to the reset controller for the eMMC capes.

Signed-off-by: Pantelis Antoniou <panto@antoniou-consulting.com>
---
 firmware/capes/BB-BONE-eMMC1-01-00A0.dts |    7 +++----
 firmware/capes/cape-bone-2g-emmc1.dts    |    7 +++----
 2 files changed, 6 insertions(+), 8 deletions(-)

diff --git a/firmware/capes/BB-BONE-eMMC1-01-00A0.dts b/firmware/capes/BB-BONE-eMMC1-01-00A0.dts
index ebac267..0776bee 100644
--- a/firmware/capes/BB-BONE-eMMC1-01-00A0.dts
+++ b/firmware/capes/BB-BONE-eMMC1-01-00A0.dts
@@ -30,9 +30,6 @@
 					0x14 0x31	/* gpmc_ad5.mmc1_dat5, INPUT_PULLUP | MODE1 */
 					0x18 0x31	/* gpmc_ad6.mmc1_dat6, INPUT_PULLUP | MODE1 */
 					0x1c 0x31	/* gpmc_ad7.mmc1_dat7, INPUT_PULLUP | MODE1 */
-
-					/* the reset */
-					0x50 0x17	/* gpmc_a4.gpio1_20, OUTPUT | MODE7 | PULLUP */
 				>;
 			};
 		};
@@ -46,8 +43,10 @@
 			bus-width = <8>;
 			ti,non-removable;
 			status = "okay";
-			reset-gpios = <&gpio2 20 1>;	/* active low */
 			ti,vcc-aux-disable-is-sleep;
+
+			reset = <&rstctl 0 0>;
+			reset-names = "eMMC_RSTn";
 		};
 	};
 };
diff --git a/firmware/capes/cape-bone-2g-emmc1.dts b/firmware/capes/cape-bone-2g-emmc1.dts
index 69bb684..bf26ae1 100644
--- a/firmware/capes/cape-bone-2g-emmc1.dts
+++ b/firmware/capes/cape-bone-2g-emmc1.dts
@@ -31,9 +31,6 @@
 					0x14 0x31	/* gpmc_ad5.mmc1_dat5, INPUT_PULLUP | MODE1 */
 					0x18 0x31	/* gpmc_ad6.mmc1_dat6, INPUT_PULLUP | MODE1 */
 					0x1c 0x31	/* gpmc_ad7.mmc1_dat7, INPUT_PULLUP | MODE1 */
-
-					/* the reset */
-					0x50 0x17	/* gpmc_a4.gpio1_20, OUTPUT | MODE7 | PULLUP */
 				>;
 			};
 		};
@@ -47,8 +44,10 @@
 			bus-width = <8>;
 			ti,non-removable;
 			status = "okay";
-			reset-gpios = <&gpio2 20 1>;	/* active low */
 			ti,vcc-aux-disable-is-sleep;
+
+			reset = <&rstctl 0 0>;
+			reset-names = "eMMC_RSTn";
 		};
 	};
 };
-- 
1.7.10.4

